docs/Producing.md

```diff
@@ -36,7 +36,7 @@ The `PRODUCER` block defines:
 
 The serializers can have the following value:
 
-   * `string`: similar serializer to the Kafka Java client [org.apache.kafka.common.serialization.StringSerializer](https://github.com/apache/kafka/blob/master/clients/src/main/java/org/apache/kafka/common/serialization/StringSerializer.java) which currently only supports `UTF-8` encoding.
+   * `string`: similar serializer to the Kafka Java client [org.apache.kafka.common.serialization.StringSerializer](https://github.com/apache/kafka/blob/master/clients/src/main/java/org/apache/kafka/common/serialization/StringSerializer.java). By default it supports `UTF-8` encoding, but you can specify the encoding as parameter like this `string(base64)`. The valid encoding value are defined in the [Buffers and character encodings](https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings).

```

Comment:
The valid encoding values are defined in [Node.js' buffers and character encodings]